cards/src/main/resources/cards/custom/group4/wraith/classic/minion_grave_carer_yorn.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "Grave-Carer Yorn",
+  "baseManaCost": 7,
+  "type": "MINION",
+  "heroClass": "CRIMSON",
+  "baseAttack": 4,
+  "baseHp": 4,
+  "rarity": "LEGENDARY",
+  "description": "Your Skeletons have +1/+1 and <b>Guard</b>. <b>Opener</b>: Summon four 1/1 Skeletons.",
+  "battlecry": {
+    "targetSelection": "NONE",
+    "spell": {
+      "class": "MetaSpell",
+      "spells": [
+        {
+          "class": "SummonSpell",
+          "boardPositionRelative": "RIGHT",
+          "cards": [
+            "token_skeleton_wraith",
+            "token_skeleton_wraith"
+          ]
+        },
+        {
+          "class": "SummonSpell",
+          "boardPositionRelative": "LEFT",
+          "cards": [
+            "token_skeleton_wraith",
+            "token_skeleton_wraith"
+          ]
+        }
+      ]
+    }
+  },
+  "auras": [
+    {
+      "class": "BuffAura",
+      "target": "FRIENDLY_MINIONS",
+      "attackBonus": 1,
+      "hpBonus": 1,
+      "filter": {
+        "class": "AttributeFilter",
+        "attribute": "SKELETON",
```

Comment:
it's a lot easier to do it this way, there's going to be minions with the word "skeleton" in the name, so giving them the attribute works out to make everything easier? also I don't know about skeleton tokens with taunt, whether a new token needs to be made or existing ones can be given taunt? in the prior scenario its just easier to have a SKELETON attribute, and saves cards having to be modified over the time because a new skeleton was added and its "specific card filter" as apposed to attribute based